docs/move/overview.md

```diff
@@ -39,7 +39,7 @@ Move modules define the rules for updating the global state of the Libra Blockch
 - The key feature of Move is the ability to define custom resource types. Resource types are used to encode safe digital assets with rich programmability.
 - Resources are ordinary values in the language. They can be stored as data structures, passed as arguments to procedures, returned from procedures, and so on.
 - The Move type system provides special safety guarantees for resources. Move resources can never be duplicated, reused, or discarded. A resource type can only be created or destroyed by the module that defines the type. These guarantees are enforced statically by the [Move virtual machine](reference/glossary.md#move-virtual-machine-mvm) via bytecode verification. The Move virtual machine will refuse to run code that has not passed through the bytecode verifier.
-- All Libra currencies are implemented using the generic `Libra::T type`. For example: the LBR currency is represented as `Libra::T<LBR::T>` and a hypothetical USD currency would be represented as `Libra::T<USD::T>`. `Libra::T` has no special status in the language; every Move resource enjoys the same protections.
+- All Libra currencies are implemented using the generic `Libra` type. For example: the LBR currency is represented as `Libra<LBR>` and a hypothetical USD currency would be represented as `Libra<USD>`. `Libra` has no special status in the language; every Move resource enjoys the same protections.
```

Comment:
In the site preview, I can't seem to navigate to this content? How do I get there from the main page with cards?

Reply:
If you click on Move in the left nav it will take you here.